Library/Formula/musicbox.rb

```diff
@@ -0,0 +1,15 @@
+class MusicBox < Formula
+  desc "A concise command line interface musicbox"
+  homepage "https://github.com/darknessomi/musicbox"
+  url "https://pypi.python.org/packages/source/N/NetEase-MusicBox/NetEase-MusicBox-0.1.6.5.tar.gz#md5=f731f7c94804a42d2acde2825dba37e1"
```

Comment:
Please cut the `#md5=f731f7c94804a42d2acde2825dba37e1` bit off the URL. At that point it should understand the version as well.